src/main/java/org/opentripplanner/graph_builder/module/ned/ElevationModule.java

```diff
@@ -157,32 +177,30 @@ public void buildGraph(Graph graph, GraphBuilderModuleSummary graphBuilderModule
         );
         log.info(setElevationsFromDEMTask.start());
 
-        AtomicInteger nProcessed = new AtomicInteger();
+        // Multithread elevation calculations
+        ForkJoinPool forkJoinPool = new ForkJoinPool();
 
-        LinkedList<StreetWithElevationEdge> edgesToCalculate = new LinkedList<>();
+        // For unknown reasons, the interpolation of heights at coordinates is a synchronized method in the commonly
+        // used Interpolator2D class. Therefore, it is critical to use a dedicated Coverage instance for each thread to
+        // avoid other threads waiting for a lock to be released on the Coverage instance. This concurrent HashMap will
+        // store these thread-specific Coverage instances.
+        ConcurrentHashMap<Long, Coverage> coveragesForThread = new ConcurrentHashMap<>();
+
+        List<StreetWithElevationEdge> streetWithElevationEdges = new LinkedList<>();
```

Comment:
Is there a typo in this variable name? Should it be `streetsWithElevationEdges`?